### ISSUE TITLE
Improper SR Label for Dropdown Menu

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1050]
- [x] Explore Data > Compare rates of x between y and z	SR	Improper announcement for list items on disease dropdown menu.	"ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64) | VoiceOver
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS
Window | Firefox Version: 87.0 (64-bit) | JAWS
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
1) Open this Url: https://healthequitytracker.org/ 
2) Navigate to the explore the data tab and select.
3) Navigate to the Investigate rates (disease) popup button and select.
4) Navigate through the popup dialog to observe this issue.
ACTUAL RESULT: Improper announcement for list items. Screen reader is announcing all list items as button instead of a list item.
EXPECTED RESULT: Provide proper label for menu list items instead of button."	Video: https://drive.google.com/file/d/1Kcc5sSVY70HAz9QsDTqigon8RboW93SL/view?usp=sharing

- [x] Explore Data > Compare rates of x between y and z	SR	Selected status of item is not verbalize to the screen reader user.	"ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64) | VoiceOver
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | JAWS
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
1) Open this Url: https://healthequitytracker.org/ 
2) Navigate to the explore the data tab and select.
3) Navigate to the Investigate rates (disease) popup button and select.
4) Navigate through the popup dialog to observe this issue.
ACTUAL RESULT: Selected status of item is not verbalize to the screen reader user
EXPECTED RESULT: Selected status of items should be verbalize to the screen reader user "	Video: https://drive.google.com/file/d/1Kcc5sSVY70HAz9QsDTqigon8RboW93SL/view?usp=sharing		Created new issue for list items on row 7

- [x] Explore Data > Compare rates of x between y and z	SR	Improper label is associated with disease button.	"ENVIRONMENT:
OS | Browser | Screen Reader
macOS Big Sur:11.4|Version 90.0.4430.212 (Official Build) (x86_64) | VoiceOver
ChromeBook | Chrome Version: 90.0.4430.218 (Official Build) (64-bit) | ChromeVox
Window | Chrome Version 91.0.4472.77 (Official Build) (64-bit) | JAWS/NVDA
Window | Firefox Version: 87.0 (64-bit) | JAWS
PREREQUISITE:
Enable the respective screen reader.
STEPS TO REPRODUCE:
1) Open this Url: https://healthequitytracker.org/ 
2) Navigate to the explore the data tab and select.
3) Navigate to the Investigate rates (disease) popup button to observe this issue.
ACTUAL RESULT: Disease dropdown is improperly labeled with default selected list item ' covid-19 button'. 
EXPECTED RESULT: Predefined label should be associated with the Disease listbox Instead announcing the selected Disease as label(COVID-19)."	VIDEO: https://drive.google.com/file/d/109kWRfHIyosRSzn5Xn6NuaJM0ya3Vq9W/view?usp=sharing	Same issue is observed for city and state buttons.	done

**Comment by benhammondmusic** [comment]
Holding off on this fix until we finalize design of new dropdown selector with added variables. Possible we need to use role="menu"; or perhaps simply better aria-labels
